README.md

```diff
@@ -106,6 +106,7 @@ As a JavaScript React component, the project is designed to be used in two diffe
 The following flags are available to toggle experimental features:
 
 - `oldgraph` - From release v3.8.0. Display old version of graph (dagre algorithm) without improved graphing algorithm. (default `false`)
+- `parameters` - Hides parameters by default on page load. When parameters are hidden, functions using parameters are highlighted. And when parameters are visible, they are highlighted. 
```

Comment:
I would change the wording to something along this line for simplicity: 
 `From release v3.12.0. Enables new behaviour of parameter nodes, which includes disabling parameter nodes by default, and will be highlighted on toggling their visibility. (default `false`)`

Reply:
> @rashidakanchwala saw that you pushed the latest changes for the flag behaviour to this PR - please ensure that the feature branch for #416 is up to date and maintained :)
> 
> To make it easier, I would say to focus on getting #416 off to the finish line and merged to main first before making any future changes on this branch.

Yes, because this feature is not relevant to 2480. Hence i don't have to push it there.

Reply:
> I would change the wording to something along this line for simplicity:
> `From release v3.12.0. Enables new behaviour of parameter nodes, which includes disabling parameter nodes by default, and will be highlighted on toggling their visibility. (default false)`

@studioswong -- reworded it again to make it a bit shorter now. From release v3.12.0. Disable parameters on page load and highlight parameter connections.